cmake/bundle/macOS/bundleutils.cmake

```diff
@@ -117,13 +117,13 @@ if(EXISTS
   execute_process(
     COMMAND
       /usr/bin/codesign --remove-signature
-      "${CMAKE_INSTALL_PREFIX}/${_BUNDLENAME}/Contents/Frameworks/Sparkle.framework/Versions/A/Resources/Autoupdate.app"
+      "${CMAKE_INSTALL_PREFIX}/${_BUNDLENAME}/Contents/Frameworks/Sparkle.framework/Versions/B/Autoupdate"
```

Comment:
I assume this could be removed entirely since there's no app bundle to (re-)sign anymore?